components/gitpod-cli/cmd/ports-list.go

```diff
@@ -30,7 +29,7 @@ var listPortsCmd = &cobra.Command{
 		ports, portsListError := supervisor_helper.GetPortsList(ctx)
 
 		if portsListError != nil {
-			log.WithError(portsListError).Error("Could not get the ports list.")
+			utils.LogError(ctx, portsListError, "Could not get the ports list.")
```

Comment:
Does it mean it can't be tested, @iQQBot?
If so, can we merge it by just reviewing the code and confirming (on Preview Environment) it's not breaking anything?

Reply:
@iQQBot, for context we started here because we had another issue but eventually we'll add it to all errors

Reply:
> we started here

I meant in this file. Because of https://github.com/gitpod-io/gitpod/pull/13680